src/main/java/org/jabref/Launcher.java

```diff
@@ -20,6 +20,7 @@
 ///
 /// - Handle the command line arguments
 /// - Start the JavaFX application (if not in CLI mode)
+
```

Comment:
Do not add empty line here. JavaDoc should be close to class

Reply:
Made the changes.